pkg/controller/storagecluster/reconcile.go

```diff
@@ -1152,3 +1156,134 @@ func getMonCount(nodeCount int) int {
 
 	return count
 }
+
+// ensureJobTemplates ensures if the osd removal job template exists
+func (r *ReconcileStorageCluster) ensureJobTemplates(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	osdCleanUpTemplate := &openshiftv1.Template{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      sc.Name + "-osd-removal",
+			Namespace: sc.Namespace,
+		},
+	}
+	_, err := controllerutil.CreateOrUpdate(context.TODO(), r.client, osdCleanUpTemplate, func() error {
+		osdCleanUpTemplate.Objects = []runtime.RawExtension{
+			{
+				Object: newCleanupJob(sc),
+			},
+		}
+		osdCleanUpTemplate.Parameters = []openshiftv1.Parameter{
+			{
+				Name:     "ROOK_IMAGE",
+				Required: true,
+				Value:    os.Getenv("ROOK_CEPH_IMAGE"),
```

Comment:
- This is an env var to the ocs operator? 
- Instead of the parameter, let's just set the image directly in the template spec. For example, on lines 1237 and 1256 just set it directly instead of setting it in the param here.

Reply:
yes.